arangod/Cluster/ClusterInfo.cpp

```diff
@@ -4831,22 +4822,21 @@ std::unordered_map<ServerID, std::string> ClusterInfo::getServers() {
     loadServers();
   }
   READ_LOCKER(readLocker, _serversProt.lock);
-  std::unordered_map<ServerID, std::string> serv = _servers;
-  return serv;
+  return _servers;
 }
 
 std::unordered_map<ServerID, std::string> ClusterInfo::getServerAliases() {
-  READ_LOCKER(readLocker, _serversProt.lock);
   std::unordered_map<std::string, std::string> ret;
+  READ_LOCKER(readLocker, _serversProt.lock);
   for (const auto& i : _serverAliases) {
     ret.try_emplace(i.second, i.first);
```

Comment:
`std::unordered_map<ServerID, std::string> _serverAliases;` why not just use the copy constructor?

Reply:
I tried it, and then I got test failures... :smile:
I overlooked that this funciton is returning the map in reversed order, i.e. not `{ key => value }` but `{ value => key }`. So just copying doesn't work. The map has to be rebuilt, also because of potential duplicates.
I don't know _why_ this function returns stuff in reverse order, but if I change it, the resilience tests go nuts.